depobs/worker/tasks.py

```diff
@@ -395,26 +395,24 @@ def get_github_advisories(package_name: str) -> None:
 
     query = (
         """
-    {
-        securityVulnerabilities(ecosystem: NPM, first: 100, package: \""""
-        + package_name
-        + """\", orderBy: {field: UPDATED_AT, direction: DESC}) {
-            nodes {
-                advisory {
+    {{
+        securityVulnerabilities(ecosystem: NPM, first: 100, package: {0}, orderBy: {{field: UPDATED_AT, direction: DESC}}) {{
+            nodes {{
+                advisory {{
                     id, description, permalink, publishedAt, severity, summary, updatedAt, withdrawnAt
-                }
-                package {
+                }}
+                package {{
                     name
-                }
-            }
-            pageInfo {
+                }}
+            }}
+            pageInfo {{
                 endCursor, hasNextPage, hasPreviousPage, startCursor
-            }
+            }}
             totalCount
-        }
-    }
+        }}
+    }}
     """
-    )
```

Comment:
nit: we're on python 3.8 so we can use an f-string https://www.python.org/dev/peps/pep-0498/#examples-from-python-s-source-code